py/kubeflow/kfctl/testing/pytests/kf_is_ready_test.py

```diff
@@ -62,16 +61,29 @@ def test_katib_is_ready(record_xml_attribute, namespace):
 
   util.load_kube_config()
 
+  for deployment_name in deployments:
+    logging.info("Verifying that deployment %s started...", deployment_name)
+    util.wait_for_deployment(api_client, namespace, deployment_name, 10)
+
+def test_katib_is_ready(record_xml_attribute, namespace):
```

Comment:
Do you want to add back pytest.mark.xfail?

Reply:
Verified with @richardsliu that this test should pass now so that i removed the mark.